src/main/java/com/peerrequest/app/api/CategoriesController.java

```diff
@@ -12,41 +12,45 @@ public class CategoriesController {
 
     @GetMapping(value = "/categories", produces = "application/json")
     public List<Category> getCategories() {
-        return List.of(mockCategory());
+        return List.of(mockUpCategory);
     }
 
     @GetMapping(value = "/categories/{categoryId}", produces = "application/json")
     public Category getCategory(@PathVariable final int categoryId) {
-        return mockCategory();
+        return mockUpCategory;
     }
 
     @PostMapping(value = "/categories", produces = "application/json")
     public Category createCategory() {
-        return mockCategory();
+        return mockUpCategory;
     }
 
     @PatchMapping(value = "/categories/{categoryId}", consumes = "application/json", produces = "application/json")
-    public Category partialUpdateCategory(@RequestBody final Category.CategoryUpdater categoryUpdater) {
-        return mockCategory();
+    public Category patchCategory(@RequestBody final Category.CategoryUpdater categoryUpdater) {
+        mockCategory().setName(categoryUpdater.name());
```

Comment:
Patch doesn't work because with every call of mockCategory() a new category is created, so the name wont't be updated. Use mockUpCategory.setName() and so on instead.